app/src/features/Batches/BatchList.tsx

```diff
@@ -146,65 +126,67 @@ const BatchList = (): JSX.Element => {
     );
   return (
     <>
-      <div className={classes.accordions}>
+      <div className={classes.batchList}>
         {batches?.results &&
           batches.results.map((batch) => (
-            <Accordion className={classes.accordion} key={`batch-${batch.id}`}>
-              <AccordionSummary
-                classes={{
-                  root: classes.accordionSummary,
-                  content: classes.accordionSummaryContent,
-                }}
-                expandIcon={<ExpandMore />}
-                aria-controls="panel1a-content"
-                id="panel1a-header"
-              >
-                <div className={classes.accordionSummaryTitle}>
-                  <div>
-                    <Typography variant="subtitle2">
-                      {!batch.completed_at &&
-                        !batch.canceled_at &&
-                        t("batchInProgress")}
-                      {batch.completed_at &&
-                        batch.errors.length > 0 &&
-                        t("batchErrors", {
-                          errors: batch.errors.length.toString(),
-                        })}
-
-                      {batch.completed_at &&
-                        batch.errors.length === 0 &&
-                        t("batchSuccess")}
-                      {batch.canceled_at && t("batchCanceled")}
-                    </Typography>
-                    <Typography className={classes.batchId}>
-                      {batch.id}
-                    </Typography>
-                  </div>
+            <Paper
+              key={batch.id}
+              className={classes.listItem}
+              variant="outlined"
+              elevation={2}
+            >
+              <div>
+                <div className={classes.title}>
                   <Typography variant="subtitle2">
-                    {new Date(batch.created_at)
-                      .toLocaleString()
-                      .split(",")
-                      .join(" -")}
+                    {!batch.completed_at &&
+                      !batch.canceled_at &&
+                      t("batchInProgress")}
+                    {batch.completed_at && batch.errors.length > 0 && (
+                      <>
+                        {t("batchErrors", {
+                          count: batch.errors.length,
+                        })}{" "}
+                        <Link
+                          target="_blank"
+                          rel="noopener"
+                          href={`${KIBANA_URL}app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-1y,to:now))&_a=(columns:!(_source),filters:!(),interval:auto,query:(language:kuery,query:${batch.id}),sort:!())`}
```

Comment:
doing this directly in the JSX is very heavy.
define a function `createKibanaLink()` external to the component instead.

Reply:
also, I asked the ops team how they're searching for errors and this does seem to be the right query.
can we try something else instead of the batchId ?
I think the query can juste be "error" and the start time should be batch.created_at